CONTRIBUTING.md

```diff
@@ -1,303 +1,61 @@
+#### 📝 Protip:
 
-## Community Guidelines
+Your first issue can be a fixing a spelling mistake or a grammatical error. If want to practice the workflow of contributing try [first-contributions](https://github.com/firstcontributions/first-contributions)
 
-We want to keep the IPFS community awesome, growing and collaborative. We need your help to keep it that way. To help with this we've come up with some general guidelines for the community as a whole:
+_watch our video which will take you step by step in how to submit a patch to IPFS_
 
-- Be nice: Be courteous, respectful and polite to fellow community members: no regional, racial, gender, or other abuse will be tolerated. We like nice people way better than mean ones!
-- Encourage diversity and participation: Make everyone in our community feel welcome, regardless of their background and the extent of their contributions, and do everything possible to encourage participation in our community.
-- Keep it legal: Basically, don't get anybody in trouble. Share only content that you own, do not share private or sensitive information, and don't break laws.
-- Stay on topic: Make sure that you are posting to the correct channel and avoid off-topic discussions. Remember when you update an issue or respond to an email you are potentially sending to a large number of people. Please consider this before you update. Also remember that nobody likes spam.
 
-There is also a more extensive [code-of-conduct](code-of-conduct.md).
+###  👷🏽‍♀️ Architecture of JS-IPFS
 
-## Dive Right In
+[Overview of js-ipfs Architecture](https://github.com/ipfs/js-ipfs/blob/master/img/overview.png)
 
-If you're curious to hack on IPFS right now and you just need an issue to focus on, check out [this search](https://github.com/search?utf8=%E2%9C%93&q=label%3A%22difficulty%3Aeasy%22+label%3A%22help+wanted%22+user%3AIPFS+is%3Aopen+&type=Issues) for issues tagged as "help wanted". Generally, these should be easier for newcomers, and are great places to start hacking away. Have fun!
+_watch our video which gives you a tour of the js-ipfs stack_
 
-## Reporting Issues
 
-If you find bugs, mistakes, inconsistencies in the IPFS project's code or documents, please let us know by filing an issue at the appropriate issue tracker (we use multiple repositories). No issue is too small.
+### 🐣 New to Git:
 
-The main issues for bug reporting are as follows:
-- [go-ipfs/issues](https://github.com/ipfs/go-ipfs/issues) - Issues related to running the Go implementation of IPFS.
-- [js-ipfs/issues](https://github.com/ipfs/js-ipfs/issues) - Issues related to running the js implementation of IPFS.
-- [notes/issues](https://github.com/ipfs/notes/issues) - For general notes and ideas.
-- [specs/issues](https://github.com/ipfs/specs/issues) - For protocol discussions.
+If you are completely new to git then run through these tutorials and get a feel of how git works.
```

Comment:
Since this section is for people new to git, should we add a sentence before this one explaining succinctly what git is and how it helps us work together as a community?